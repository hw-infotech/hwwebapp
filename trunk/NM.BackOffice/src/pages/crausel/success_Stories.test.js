import {
  fireEvent,
  getAllByRole,
  getByTestId,
  getByText,
  render,
  screen,
  within,
} from "@testing-library/react";
import Success_Stories from "./success_Stories";
import userEvent from "@testing-library/user-event";

describe("Expand body button", () => {
  it("should render OrderModuleBody when clicked", () => {
    render(<Success_Stories />);
    const button = screen.getByTestId("butoon1");
    userEvent.click(button);
    expect(screen.getByText("Edit")).toBeInTheDocument();
  });
});
