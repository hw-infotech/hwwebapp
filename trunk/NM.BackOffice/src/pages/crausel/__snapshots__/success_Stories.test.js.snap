// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test success stories component button 1`] = `
<div
  class="breadecumbs_box "
>
  <ul
    class="breadcumbs_style"
  >
    <li>
      Dashboard
       
      &gt;
    </li>
    <li>
      Sliders/Carousels
       
      &gt;
    </li>
    <li
      style="color: rgb(115, 126, 210);"
    >
      Success Stories
    </li>
  </ul>
</div>
`;
